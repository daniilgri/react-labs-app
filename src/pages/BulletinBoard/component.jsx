import React from "react";
import Board from "../../components/related/BulletinBoard/Board";
import SearchBar from "../../components/related/BulletinBoard/SearchBar";
import Header from "../../components/global/Header";
import { Wrapper } from "./styles";

const component = () => (
  <>
    <Header />
    <SearchBar />
    <Wrapper>
      <Board />
    </Wrapper>
  </>
);

export default component;
