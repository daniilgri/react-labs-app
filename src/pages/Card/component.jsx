import React from "react";
import { Wrapper } from "./styles";

const component = () => <Wrapper></Wrapper>;

export default component;
