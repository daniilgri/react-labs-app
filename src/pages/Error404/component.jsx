import React from "react";
import { Wrapper } from "./styles";

const component = () => <Wrapper />;

export default component;
