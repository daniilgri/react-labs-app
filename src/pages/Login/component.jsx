import React from "react";

import Header from "../../components/global/Header";
import LoginForm from "../../components/related/Login/LoginForm";
import { Wrapper } from "./styles";

const component = () => (
  <>
    <Header />
    <Wrapper>
      <LoginForm />
    </Wrapper>
  </>
);

export default component;
