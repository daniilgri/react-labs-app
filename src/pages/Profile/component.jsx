import React from "react";

import PersonalForm from "../../components/related/Profile/PersonalForm";
import PasswordChangeForm from "../../components/related/Profile/PasswordChangeForm";
import RequestOnDeleteSection from "../../components/related/Profile/RequestOnDeleteSection";
import Header from "../../components/global/Header";
import { Wrapper } from "./styles";

const component = () => (
  <React.Fragment>
    <Header />
    <Wrapper>
      <PersonalForm />
      <PasswordChangeForm />
      <RequestOnDeleteSection />
    </Wrapper>
  </React.Fragment>
);

export default component;
