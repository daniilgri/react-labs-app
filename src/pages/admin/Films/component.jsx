import React from "react";
import FilmsTable from "../../../components/related/admin/Films/FilmsTable";
import { Wrapper } from "./styles";

const component = () => (
  <React.Fragment>
    <Wrapper>
      <FilmsTable />
    </Wrapper>
  </React.Fragment>
);

export default component;
