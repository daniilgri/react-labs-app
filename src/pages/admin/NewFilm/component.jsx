import React from "react";

import AdminHeader from "../../../components/global/AdminHeader";
import Form from "../../../components/related/admin/NewFilm/Form";
import { Wrapper } from "./styles";

const component = () => (
  <React.Fragment>
    <AdminHeader />
    <Wrapper>
      <Form />
    </Wrapper>
  </React.Fragment>
);

export default component;
