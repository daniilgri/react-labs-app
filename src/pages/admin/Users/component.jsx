import React from "react";
import AdminHeader from "../../../components/global/AdminHeader";
import UsersTable from "../../../components/related/admin/Users/UsersTable";
import { Wrapper } from "./styles";

const component = () => (
  <>
    <AdminHeader />
    <Wrapper>
      <UsersTable />
    </Wrapper>
  </>
);

export default component;
