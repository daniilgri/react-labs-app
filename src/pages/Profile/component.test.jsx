import React from "react";
import renderer from "react-test-renderer";

import component from "./component";

describe("Profile component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<component />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
