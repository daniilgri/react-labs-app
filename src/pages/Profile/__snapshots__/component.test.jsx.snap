// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile component renders correctly 1`] = `<component />`;
